README.md

```diff
@@ -116,6 +117,7 @@ Apache 2.0 license, except for contributions copied from Vim (identified by the
 
 [license-commit]: https://github.com/neovim/neovim/commit/b17d9691a24099c9210289f16afb1a498a89d803
 [nvim-features]: https://neovim.io/doc/user/vim_diff.html#nvim-features
+[nvim-news]: https://neovim.io/doc/user/news
```

Comment:
Docs are generated with `*.html` suffix, because legacy. Would be cool to avoid that (need redirects for old links), requires some sort of configuration I think in jekyll? https://github.com/neovim/neovim.github.io/

```suggestion
[nvim-news]: https://neovim.io/doc/user/news.html
```